README.md

```diff
@@ -162,10 +166,19 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [RethinkDb.Driver](https://github.com/bchavez/RethinkDb.Driver) - C#/.NET RethinkDB driver with 100% ReQL API coverage.
 * [tarantool-csharp](https://github.com/progaudi/tarantool-csharp) - .NET CLI client for Tarantool NoSql database.
 
+### Datetime
+* [Exceptionless.DateTimeExtensions](https://github.com/exceptionless/Exceptionless.DateTimeExtensions) - DateTimeRange, Business Day and various DateTime, DateTimeOffset, TimeSpan extension methods.
+
+### Distributed Computing
```

Comment:
Actually, we don't have many items for **distributed computing** on this list so I think we should put it into the **Misc**